bin/push-iptables-image

```diff
@@ -26,15 +27,19 @@ set -o pipefail
 
 PROJECT="istio-testing"
 DATE_PART=$(date +"%Y%m%d")
-SHA_PART=$(git show -q HEAD --pretty=format:%h)
+SHA_PART=$(git show HEAD --pretty=format:%h | head -1 || /bin/true)
 DOCKER_TAG="${DATE_PART}-${SHA_PART}"
-IMAGE_NAME="gcr.io/${PROJECT}/iptables:${DOCKER_TAG}"
+IMAGE_NAME_UBUNTU="docker.io/${PROJECT}/iptables_ubuntu:${DOCKER_TAG}"
```

Comment:
we should make the hub a variable but not just change from gcr.io/istio-testing to docker.io/istio-testing

cc @sebastienvas

Reply:
agreed.